blog/_posts/2016-03-28-4-ways-to-find-a-team.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 4 Ways to Find a Team for RGSoC 2016
+layout: post
+created_at: Mon Mar 28 2016
+permalink: blog/2016-03-28-4-ways-to-find-a-team
+current: blog
+author: Ana Sofia
+twitter: RailsGirlsSoC
+---
+
+![4 Ways to Find a Team for RGSoC 2016](/img/blog/2016/RGSoC-finding-a-team.png)
+
+<div style="text-align:right"><font color="grey"><small><i>Team Rubyherzlein @RGSoC 2015</i></small></font></small></div>
```

Comment:
can you remove the `div-style` part in the caption? aligning it to the right was an exception for the illustration in one of the last posts, because in that case it looked better, but we usually always stick to left-alignment.